src/lib/functions/server.js

```diff
@@ -105,6 +106,20 @@ const createHandler = function ({ functionsRegistry }) {
       const { error } = await func.invoke(event, clientContext)
 
       handleBackgroundFunctionResult(functionName, error)
+    } else if (await func.isScheduled()) {
+      if (event.headers['user-agent'] !== 'Netlify Clockwork') {
+        return response
+          .status(400)
+          .send(
+            `Scheduled function cannot be requested via HTTP. Invoke using 'netlify functions:invoke ${functionName}' instead.`,
```

Comment:
[dust] Will this help make it clearer that the suggested solution is a CLI command?

```suggestion
            `Scheduled function cannot be requested via HTTP. Invoke using the 'netlify functions:invoke ${functionName}' command instead.`,
```

Reply:
https://github.com/netlify/cli/pull/3689/commits/01c986a691619e3a8538fe2e71445153551bc1a7